src/ui/ResultList/ResultList.ts

```diff
@@ -589,7 +596,9 @@ export class ResultList extends Component {
 
       if (this.options.enableInfiniteScroll && results.results.length == data.queryBuilder.numberOfResults) {
         // This will check right away if we need to add more results to make the scroll container full & scrolling.
-        this.scrollBackToTop();
+        if (this.options.enableScrollToTop) {
```

Comment:
Now that the logic is more than just a line, I would extract it into a decidated method, and reuse it here and on line 619. e.g.
```
private scrollToTopIfEnabled() {
 this.options.enableScrollToTop && this.scrollBackToTop();
}
```

Alternatively, since the `scrollBackToTop` method is private, you could rename it and add the condition inside it.

```
private scrollBackToTopIfEnabled() {
    if (!this.options.enableScrollTop) {
        return;
    }
   
    // old logic as before
}
```

Reply:
Good Idea.
I will rename the function and add the if